polynote-server/src/main/scala/polynote/server/Server.scala

```diff
@@ -30,6 +30,7 @@ class Server(kernelFactory: Kernel.Factory.Service) extends polynote.app.App wit
 
   override def run(args: List[String]): ZIO[Environment, Nothing, Int] = for {
     args     <- ZIO.fromEither(Server.parseArgs(args)).orDie
+    _        <- Logging.info(s"Loading configuration from ${args.configFile}")
```

Comment:
It'd be useful if there was a way to see what the config resolved to as well. 

Ideally:
1. Somewhere in the UI you'd be able to see information about the server such as config location, config value, etc. 
2. Log of `PolynoteConfig` (to debug startup issues where the UI isn't accessible).